samples/streaming/consumer/src/main.rs

```diff
@@ -39,7 +39,12 @@ async fn stream_images(
     // The stream is infinite, so we will just take number_of_images elements and then disconnect.
     let mut stream = stream.take(number_of_images);
     while let Some(item) = stream.next().await {
-        let media_content = item.unwrap().media.unwrap().media_content;
+        let opt_media = item.unwrap().media;
+        if opt_media.is_none() {
+            warn!("No media value present in message");
+            break;
```

Comment:
this should probably be `continue` to go to the next item, this will stop processing altogether